### PR TITLE
correct conversion of 0-9 keyCodes

### LINKED ISSUE #30: Number pad digit keys not handled properly
When passed through the `normalizedKeyForEvent` function, key events for number pad digits are incorrectly transformed because the `keyIdentifier` property for the event has the same Unicode value as `a-i`.

The `keyIdentifier` property is deprecated and non-standard ([API](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier)), so this check should be removed, or at least placed below the check for the `keyCode` property.

If the `transformKeyIdentifier` check is put below the `transformKeyCode` check, an issue with the `transformKeyCode` function becomes apparent when testing number keys. The `keyCode` parameter should have `48` and `96` subtracted **from** it for top row numeric keys and number pad digits respectively, not the other way around.

I've created a pull request for this issue.

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #30 by converting 0-9 keyCodes to the correct number

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
@cdata kindly review

File: test/basic-test.html [hunk 1]
```diff
@@ -273,6 +273,20 @@
...
+      test('top row can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 49;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
+      });
+
+      test('number pad digits can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 97;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
```
Comment (cgrail) [inline test/basic-test.html]: You could extract a helper function for some of this tests like:
```javascript
function expectKeyboardEventMatchesKeys(keyCode){
  var event = new CustomEvent('keydown');
  event.keyCode = keyCode;
  expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
}
```

and rewrite your unit test to something like that
```javascript
test('number pad digits can be done imperatively', function() {
  expectKeyboardEventMatchesKeys(97);
});
```

Comment (valdrinkoshi) [inline test/basic-test.html]: We set the `keyCode` only in these 2 tests, in the others we set `key`.
Also, the new function would rely on the existence of `keys` and hide a check :p

**Comment by valdrinkoshi** [comment]
@btelles ping should be for @cdata who's reviewing this :)

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**